FinalProject/FinalProject/src/main/java/com/globant/FinalProject/shopping/Buy.java

```diff
@@ -0,0 +1,34 @@
+package com.globant.FinalProject.shopping;
+
+/**
+ * This class represents a buy that is aggregate into the shopping cart.
+ * Just contain idProduct and quantity to buy.
+ * @author Herminio Andres Hernandez
+ * @version 05/08/2018
+ */
+public class Buy {
```

Comment:
This seems to be a DTO, when thats the case is nice to use that in the name, to easily spot that this doesn't belongs to the data model but is used only as transport